dist/gitlabci/.gitlab-ci.yml

```diff
@@ -26,8 +26,15 @@ stages:
   image: juampynr/drupal8ci:latest
   stage: test
   services:
-    - name: registry.gitlab.com/juampynr/drupal8-gitlab:latest
+    - name: mariadb:latest
       alias: db
+  variables:
+    MYSQL_ROOT_PASSWORD: root
+    MYSQL_DATABASE: drupal
+  script:
+    - mysql --version
```

Comment:
Are these three lines for debugging?

Reply:
Yup, specially the `script` part to make sure that the service is up and running before trying to run the Drupal install or DB import. Without it it was failing at first.